pcdsdevices/mirror.py

```diff
@@ -681,6 +676,24 @@ def format_status_info(self, status_info):
     bender_ds_enc_rms: {b_ds_enc_rms}
 """
 
+class KBOMirrorHE(KBOMirror):
+    """
+    Kirkpatrick-Baez Mirror with Bender Axes and Cooling.
+
+    1st gen Toyama designs with LCLS-II Beckhoff motion architecture.
+
+    Parameters
+    ----------
+    prefix : str
+        Base PV for the mirror.
+
+    name : str
+        Alias for the device.
+    """
+    # Cooling water flow and pressure sensors
+    cool_flow1 = Cpt(EpicsSignalRO, ':FLOW:1_RBV', kind='normal')
+    cool_flow2 = Cpt(EpicsSignalRO, ':FLOW:2_RBV', kind='normal')
+    cool_press = Cpt(EpicsSignalRO, ':PRESS:1_RBV', kind='normal')
 
```

Comment:
Needs two blank lines between classes.
```suggestion


class KBOMirrorHE(KBOMirror):
    """
    Kirkpatrick-Baez Mirror with Bender Axes and Cooling.

    1st gen Toyama designs with LCLS-II Beckhoff motion architecture.

    Parameters
    ----------
    prefix : str
        Base PV for the mirror.

    name : str
        Alias for the device.
    """
    # Cooling water flow and pressure sensors
    cool_flow1 = Cpt(EpicsSignalRO, ':FLOW:1_RBV', kind='normal')
    cool_flow2 = Cpt(EpicsSignalRO, ':FLOW:2_RBV', kind='normal')
    cool_press = Cpt(EpicsSignalRO, ':PRESS:1_RBV', kind='normal')


```

Reply:
fixed